docs/reference/cloud-event-list.mdx

```diff
@@ -81,6 +87,14 @@ The `component_scaled` event is emitted when a component is scaled up or down.
     "annotations": {
       "key": "value"
     },
+    "claims": {
+      "call_alias": null,
+      "issuer": "ACOJJN6WUP4ODD75XEBKKTCCUJJCY5ZKQ56XVKYK4BEJWGVAOOQHZMCW",
+      "tags": ["wasmcloud.com/experimental"],
```

Comment:
I'm surprised we still use this

Reply:
the experimental tag? Or the fact that `claims.tags` is present at all? Either way, it's just an example representation because it still exists in the implementation